### ISSUE TITLE
Ipsec_connection Phase Config

### ISSUE THREAD

**Issue by GigaChad2137** [oracle/terraform-provider-oci#1691]
Hello,

Is it posible to configurate `phase_one_details` and  `phase_two_details` in `oci_core_ipsec_connection_tunnel_management` or some other `oci_core`?  If not then why and how i can change that using some terraform/ansible etc code...

I wanted something like this 
```
 phase_one_details {
                custom_authentication_algorithm =  "SHA2_256"
                custom_dh_group =  "GROUP24"
                custom_encryption_algorithm =  "AES_256_CBC"
                is_custom_phase_one_config =  true
                is_ike_established =  false
                lifetime =  36000
              }
            
  phase_two_details{
    
      dh_group =  "GROUP24"
      is_custom_phase_two_config = true
      is_esp_established =  false
      is_pfs_enabled =  true
      lifetime =  3600
    }
```
 Puttng this in `oci_core_ipsec_connection_tunnel_management` give me   `can't configure a value for "phase_one_details": its value will be decided automatically based on the result of applying this configuration.`

**Comment by GigaChad2137** [comment]
I "solve" this with python script hitting API endpoint

ref:
https://docs.oracle.com/en-us/iaas/api/#/en/iaas/20160918/IPSecConnectionTunnel/UpdateIPSecConnectionTunnel

**Comment by iejones** [comment]
A bug fix to set phase one and two tunnel details will be released on November 15th, 2023

**Comment by iejones** [comment]
This fix was released with version [v5.21.0](https://github.com/oracle/terraform-provider-oci/releases/tag/v5.21.0). See phase_one_details and phase_one_details in the [documentation](https://registry.terraform.io/providers/oracle/oci/latest/docs/resources/core_ipsec_connection_tunnel_management) and the [example](https://github.com/oracle/terraform-provider-oci/blob/v5.21.0/examples/networking/ipsec_connections/ipsec_connection.tf#L127).
